Doc/library/argparse.rst

```diff
@@ -324,6 +344,8 @@ specifier.
     -h, --help  show this help message and exit
     --foo FOO   foo of the myprogram program
 
+.. versionchanged:: 3.14
+   The default value is now not simply ``os.path.basename(sys.argv[0])``.
```

Comment:
```suggestion
   The default ``prog`` value now reflects how ``__main__`` was actually executed,
   rather than always being ``os.path.basename(sys.argv[0])``.
```